inference-engine/src/gna_plugin/gna_plugin.cpp

```diff
@@ -1271,9 +1271,9 @@ GnaWaitStatus GNAPlugin::WaitFor(uint32_t request_idx, int64_t millisTimeout) {
     for (auto && outputBlobIt : request) {
         auto & outputBlob = outputBlobIt.second;
         auto & outputDesc = outputsDesc[output_idx];
-        if (!outputBlob->getTensorDesc().getLayout() == Layout::C && !outputBlob->getTensorDesc().getLayout() == Layout::NC &&
-            !outputBlob->getTensorDesc().getLayout() == Layout::CN && !outputBlob->getTensorDesc().getLayout() == Layout::NCHW &&
-            !outputBlob->getTensorDesc().getLayout() == Layout::CHW) {
+        if (!(outputBlob->getTensorDesc().getLayout() == Layout::C) && !(outputBlob->getTensorDesc().getLayout() == Layout::NC) &&
+            !(outputBlob->getTensorDesc().getLayout() == Layout::CN) && !(outputBlob->getTensorDesc().getLayout() == Layout::NCHW) &&
+            !(outputBlob->getTensorDesc().getLayout() == Layout::CHW)) {
```

Comment:
maybe use `!=` instead of `!(==)`?

Reply:
My target was to fix some warnings to reduce output size and fix real issues if found.
If it's critical I can change it.

Reply:
I understand your goal, but `!(==)` looks really ugly :)

Reply:
A little bit)  But now it's at least w/o incorrect result.
I'll update If it's showstopper or if other place should be fixed too